modules/monitoring-specifying-how-a-service-is-monitored-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-how-a-service-is-monitored-for-cluster-observability-operator_{context}"]
+= Specifying how a service is monitored for Cluster Observability Operator
+
+To use the metrics exposed by the sample service you created in the "Creating a `MonitoringStack` object for the Cluster Observability Operator" section, you must configure monitoring components to scrape metrics from the `/metrics` endpoint. 
+
+You can create this configuration by using a `ServiceMonitor` custom resource definition (CRD) that specifies how the service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. 
+The `ServiceMonitor` object requires a `Service` object. The `PodMonitor` object does not, which enables Prometheus to scrape metrics directly from the metrics endpoint exposed by a pod
+
+This procedure shows how to create a `ServiceMonitor` object for a sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with administrative permissions for the namespace.
+* You have installed the Cluster Observability Operator.
+* You have deployed the `prometheus-coo-example-app` sample service in the `ns1-coo` project.
++
+[NOTE]
+====
+The `prometheus-coo-example-app` sample service does not support TLS authentication. 
+====
+
+.Procedure
+
+. Create a YAML file named `example-coo-app-service-monitor.yaml` that contains the following `ServiceMonitor` object configuration details:
++
+[source,yaml]
+----
+apiVersion: monitoring.rhobs/v1alpha1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: prometheus-coo-example-monitor
+  name: prometheus-coo-example-monitor
+  namespace: ns1-coo
+spec:
+  endpoints:
+  - interval: 30s
+    port: web
+    scheme: http
+  selector:
+    matchLabels:
+      app: prometheus-coo-example-app
+----
++
+This defines a `ServiceMonitor` resource that scrapes the metrics exposed by the `prometheus-coo-example-app` sample service, which includes the `version` metric.
++
+[NOTE]
+====
+A `ServiceMonitor` resource in a COO namespace can only discover services in the same namespace. That is, the `namespaceSelector` field of the `ServiceMonitor` resource is always ignored.
```

Comment:
Not sure where do you get this, there is no such limitation "only discover services in the same namespace"

Reply:
Removed